Dockerfile

```diff
@@ -39,11 +39,12 @@ LABEL org.opencontainers.image.authors="FNNDSC <dev@babyMRI.org>" \
       org.opencontainers.image.source="https://github.com/FNNDSC/pfcon" \
       org.opencontainers.image.licenses="MIT"
 
+RUN conda install -y -c conda-forge msgpack-python==1.0.3 MarkupSafe==2.1.1 netifaces==0.10.9 PyYAML==6.0 wrapt==1.14.0
```

Comment:
Please specify these dependencies in an external file, such as one created by `conda env export`, see the documentation I sent to you in chat.